tests/test_rdpg_corr.py

```diff
@@ -79,5 +79,5 @@ def test_p_is_close(self):
         graphs2 = np.stack(graphs2)
         np.testing.assert_allclose(np.mean(graphs1, axis=0), P, atol=0.1)
         np.testing.assert_allclose(np.mean(graphs2, axis=0), P, atol=0.1)
-        np.testing.assert_allclose(np.mean(np.mean(graphs1, axis=0) - P), 0, atol=1e-4)
-        np.testing.assert_allclose(np.mean(np.mean(graphs2, axis=0) - P), 0, atol=1e-4)
+        # np.testing.assert_allclose(np.mean(np.mean(graphs1, axis=0) - P), 0, atol=1e-4)
+        # np.testing.assert_allclose(np.mean(np.mean(graphs2, axis=0) - P), 0, atol=1e-4)
```

Comment:
make sure to remove if you're not keeping this code

Reply:
@bdpedigo said that this was alright because of some errors that were popping up. If i should delete i will but i was told just to comment out for now in order to pass the test checks.